modules/administration-guide/pages/viewing-kubernetes-events.adoc

```diff
@@ -1,6 +1,6 @@
 [id="viewing-kubernetes-events"]
-// = Accessing Kubernetes events on OpenShift
-:navtitle: Accessing Kubernetes events on OpenShift
+// = Accessing {kubernetes} events on OpenShift
```

Comment:
Could we just call them cluster events?

Reply:
This navtitle is upstream only. It makes sense to keep Kubernetes here.
(The title is in a comment, we could even remove it)